source/quick-start.txt

```diff
@@ -0,0 +1,110 @@
+.. _entity-framework-quickstart:
+
+===========
+Quick Start
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/quick-start/overview.rst
+
+Create a MongoDB Cluster
+------------------------
+
+Set Up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To set up your Atlas Free Tier Cluster required for this guide, complete the guide on 
+:guides:`MongoDB Atlas Setup </atlas/account>`.
+
+After completing the steps in the Atlas guide, you have a new MongoDB
+cluster deployed in Atlas, a new database user, and
+:atlas:`sample datasets loaded </sample-data/>` into your cluster. You also have
+a connection string similar to the following in your copy buffer:
+
+.. code-block:: bash
+
+   "mongodb+srv://<username>:<password>@cluster0.abc.mongodb.net/?retryWrites=true&w=majority"
+
+Set Your Connection String
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following code at the command prompt to save your MongoDB connection string to an 
+environment variable. This method is safer than including your credentials in your source 
+code. 
+
+.. code-block:: bash
+
+   export MONGODB_URI='<your MongoDB URI>'
+
+.. important::
+
+   Make sure to replace the ``<username>`` and ``<password>`` sections of the connection 
+   string with the username and password of your Atlas user.
+
+Set Up Your Project
+-------------------
+
+Create the Project
+~~~~~~~~~~~~~~~~~~
+
+Create a new directory and initialize your project with the ``dotnet new`` command,
+as follows:
+
+.. code-block:: shell
+
+   mkdir entity-quickstart
+   cd entity-quickstart
+   dotnet new console
+
+.. _entity-framework-add-mongodb-dependency:
+
+Add MongoDB as a Dependency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the ``dotnet add`` command to add the {+provider-short+} to your project as a
+dependency.
+
+.. code-block:: shell
+
+   dotnet add package [todo: insert package here] 
+
+Query Your MongoDB Cluster from Your Application
+------------------------------------------------
+
+In this step, you'll use the {+provider-short+} 
+to connect to your MongoDB cluster and run a LINQ query on the sample data.
+
+Open the file named ``Program.cs`` in the base directory of your project. Copy the
+following sample code into ``Program.cs``: 
+
+.. literalinclude:: /includes/quick-start/Program.cs
+   :language: csharp
+   :dedent:
+
+This sample code runs a query against your sample dataset in MongoDB Atlas. Run it
```

Comment:
Suggestion:

I think "against" is unnecessary; I think "query your sample dataset" has the same intended message. I think it's also ok to use "Atlas" since it was used that way since the first step of the first heading ("Set Up a Free Tier Cluster in Atlas").